src/main/permissionsManager.ts

```diff
@@ -152,6 +153,12 @@ export class PermissionsManager extends JsonFileManager<PermissionsByOrigin> {
             return false;
         }
 
+        // For GPO servers, we always allow permissions since they are trusted
+        const serverHref = serverURL.href;
+        if (Config.registryData?.servers?.some((s) => parseURL(s.url)?.href === serverHref)) {
```

Comment:
Just sanity check question. At `Config.registryData?.servers?` we only have GPO servers, right?

Reply:
Correct, they are servers that come from the Windows registry. I was considering doing all predefined servers (including ones specified by the build), but I thought that might expose people to be tricked into downloading an app with a pre-configured server that has access to everything.